Makefile

```diff
@@ -20,8 +20,8 @@ DEST = website
 SPHINX_DEPENDENCIES = $(foreach lang, $(LANGS), $(lang)/Makefile)
 
 # Copy-paste the English Makefile everywhere it's needed.
-%/Makefile: en/Makefile
-	cp $< $@
+# %/Makefile: en/Makefile
+# 	cp $< $@
```

Comment:
not related, windows derp that should not be commited :wine_glass: 
removed now